### PR TITLE
Add information about linking CN= <nodeName> and kubelet hostname

### PR DESCRIPTION
**Pull request by deitch**
As discussed on Slack with @liggitt and @micahhausler .

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -10,7 +10,10 @@ notitle: true
 
 
 The kubelet is the primary "node agent" that runs on each
```
Comment (liggitt) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: this page is generated from https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L120-L138

manual changes here will be stomped the next time it is updated. if you want to make changes, also make them in the command help

**Comment by liggitt** [comment]
lgtm modulo comment on generated doc page

File: content/en/docs/reference/command-line-tools-reference/kubelet.md (hunk 1 above)
Comment (deitch) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: Oops. 

I don't mind making a PR there as well, but I am concerned about putting URLs - relative or absolute - in the help in the `server.go` file. Thoughts about that?

Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: yeah, probably not great to deep-link help URLs from CLI output

Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: @kubernetes/sig-docs-pr-reviews might have suggestions

Comment (deitch) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: I am going to update this to remove the elements here, and a separate PR (linked to this) for the docs in the source, as you suggested, but without actual links.

Comment (deitch) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: I removed the changes to the auto-generated file. PR for CLI https://github.com/kubernetes/kubernetes/pull/73256

**Comment by cody-clark** [comment]
[Node Auth Preview](https://deploy-preview-12336--kubernetes-io-master-staging.netlify.com/docs/reference/access-authn-authz/node/)

[Cert Preview](https://deploy-preview-12336--kubernetes-io-master-staging.netlify.com/docs/setup/certificates/)

File: content/en/docs/setup/certificates.md [hunk 2]
```diff
@@ -109,10 +109,12 @@ You must manually configure these administrator account and service accounts:
 | filename                | credential name            | Default CN                     | O (in Subject) |
 |-------------------------|----------------------------|--------------------------------|----------------|
 | admin.conf              | default-admin              | kubernetes-admin               | system:masters |
-| kubelet.conf            | default-auth               | system:node:`<nodename>`        | system:nodes   |
+| kubelet.conf            | default-auth               | system:node:`<nodeName>` (see note) | system:nodes   |
 | controller-manager.conf | default-controller-manager | system:kube-controller-manager |                |
 | scheduler.conf          | default-manager            | system:kube-scheduler          |                |
 
+Note: the value of `<nodeName>` for `kubelet.conf` **must** match precisely the value of the node name provided by the kubelet as it registers with the apiserver. For further details, read the [Node Authorization](/docs/reference/access-authn-authz/node/).
```
Comment (cody-clark) [inline content/en/docs/setup/certificates.md]: Thanks for the PR, @deitch! Before I approve it, could you possibly add this [Hugo Shortcode](https://kubernetes.io/docs/contribute/style/style-guide/#shortcodes) 

More specifically:

{{< note >}}
The value of `<nodeName>` for `kubelet.conf` **must** match precisely the value of the node name provided by the kubelet as it registers with the apiserver. For further details, read the [Node Authorization](/docs/reference/access-authn-authz/node/).
{{< /note >}}

Thanks!

Comment (deitch) [reply content/en/docs/setup/certificates.md]: Sure. Coming right away...

**Comment by cody-clark** [comment]
Awesome, thanks!

/lgtm
/approve

**Pull request merged**